external_kpis.css

```diff
@@ -1,77 +1,75 @@
 text {
-    font-family: sans-serif;
-    font-size: 12px;
-    fill: white;
+  font-family: sans-serif;
+  font-size: 12px;
+  fill: white;
 }
 #application_projects,
-#application_samples{
-    float: left;
-    margin: 5px 0px 0px 0px;
+#application_samples {
+  float: left;
+  margin: 5px 0 0 0;
 }
 #affiliations {
-    float: left;
-    margin: 5px 0px 0px 20px;
+  float: left;
+  margin: 5px 0 0 20px;
 }
 #delivery_times {
-    float: left;
-    /*margin-left: 30px;*/
-    /*margin-right: 10px;*/
-    margin: 0px 20px 20px 40px;
+  float: left;
+  margin: 0 20px 20px 40px;
```

Comment:
FYI, you can shorten this even further to:
```css
margin: 0 20px 40px;
```
(if only three values they specify top / sides / bottom)